### PR TITLE
add prod compose file with TLS config

### PR DESCRIPTION
**Pull request by Sispheor**

### CODE REVIEW

File: Squest/settings/development.py [hunk 1]
```diff
@@ -26,10 +28,7 @@
 DEBUG = True
 TESTING = sys.argv[1:2] == ['test']
 COLLECTING_STATIC = sys.argv[1:2] == ['collectstatic']
-ALLOWED_HOSTS = ['*']
-print("DEBUG: {}".format(DEBUG))
-print("TESTING: {}".format(TESTING))
-print("COLLECTING_STATIC: {}".format(COLLECTING_STATIC))
+ALLOWED_HOSTS = os.getenv('ALLOWED_HOSTS', '*').split(',')
```
Comment (a-belhadj) [inline Squest/settings/development.py:31]: old value was a list and your default value isn't a list. Is it normal ?

File: docs/deployment.md [hunk 2]
```diff
@@ -1,16 +1,89 @@
...
+
 The current deployment is based on Docker compose for testing the tool.
 
 Pre-requisites:
 
 - docker
 - docker-compose
 
-To test the application, run the full env docker compose file
+To run the application, execute the full env docker compose file
```
Comment (a-belhadj) [inline docs/deployment.md:12]: "the full env docker compose file"

File: docs/dev/dev-env.md [hunk 3]
```diff
@@ -36,9 +36,9 @@ The development environment is composed of 4 parts:
 
 ### Docker compose
 
-Run the Docker dev env to bring up database, message broker and other required system
+Run the Docker compose file with only required services to bring up database, message broker and other required system
 ```bash
-docker-compose -f dev-env.docker-compose.yml up
+docker-compose -f docker-compose.yml db phpmyadmin rabbitmq up
```
Comment (a-belhadj) [inline docs/dev/dev-env.md:41]:
```
docker-compose up db phpmyadmin rabbitmq
```

**Review by a-belhadj — CHANGES_REQUESTED** [review]

File: docs/dev/dev-env.md (hunk 3 above)
Comment (Sispheor) [reply docs/dev/dev-env.md]: Fixed

**Comment by Sispheor** [comment]
I'll send Ansible deployment with backup in a dedicated PR.
This one contains already a lot of changes.

**Comment by Sispheor** [comment]
Branch rebased. Ready for review

**Review by a-belhadj — APPROVED** [review]

**Pull request merged**